src/services/api.ts

```diff
@@ -1,13 +1,26 @@
 import axios from 'axios'
 import { HL7File, HL7Message } from '../types';
+import { stores } from '../stores'
+import { USER_STORE } from '../stores/userStore'
 
 // TODO config for url
 const baseURL = 'http://localhost:4040'
+const api = axios.create({
+  baseURL,
+})
+
+api.interceptors.response.use(function (response) {
+  return response;
+}, function (error) {
+  if (error.response.data == 'Unauthorized') {
```

Comment:
I'd lean towards checking if the status code is `401` rather than a string comparison against what the library we're using (passport) has decided to return with 401 responses.

Reply:
Good feedback! Resolved